src/main/java/frc/utils/Conversions.java

```diff
@@ -126,6 +126,16 @@ public static double microSecondsToSeconds(double microSeconds) {
 		return microSeconds / 1e6;
 	}
 
+	/**
+	 * Converts a time from milliseconds to seconds
+	 *
+	 * @param milliSeconds the milliseconds
+	 * @return the seconds
+	 */
+	public static double milliSecondsToSeconds(double milliSeconds) {
+		return milliSeconds / 1e3;
+	}
+
```

Comment:
You can remove it now. It's in another PR.

Reply:
i solved it but you didn't push the code

Reply:
danget